File: InvestorDashboard.Web/src/app/models/client-info.model.ts
export class ClientInfo {
    balance: number;
    bonusBalance: number;
    summary: number;
    // address: string;
    isTokenSaleDisabled: boolean;
    isEligibleForTransfer: boolean;
    thresholdExceeded: boolean;
    isInvestor: boolean;
    tokensAvailableForTransfer: number;

    constructor() {
        this.balance = 0;
        this.bonusBalance = 0;
    }
}

File: InvestorDashboard.Api/Models/DashboardModels/ClientInfoModel.cs
﻿namespace InvestorDashboard.Api.Models.DashboardModels
{
    public class ClientInfoModel
    {
        public string Address { get; set; }
        public long Balance { get; set; }
        public long BonusBalance { get; set; }
        public bool ThresholdExceeded { get; set; }
        public bool IsTokenSaleDisabled { get; set; }
        public bool IsEligibleForTransfer { get; set; }
        public bool IsInvestor { get; set; }
        public long TokensAvailableForTransfer { get; set; }
        public bool IsAdmin { get; set; }
    }
}
